tests/e2e/autoscale/02-install.yaml

```diff
@@ -11,13 +11,11 @@ spec:
   autoscaler:
     targetCPUUtilization: 50
     targetMemoryUtilization: 70
-    # Without this behavior the HPA will default to a scaledown stabilization
-    # window of 300 seconds. Tests should fail if this update is not successful.
     behavior:
       scaleDown:
-        stabilizationWindowSeconds: 15
+        stabilizationWindowSeconds: 1
       scaleUp:
-        stabilizationWindowSeconds: 25
+        stabilizationWindowSeconds: 1
```

Comment:
what is the motivation of setting both of these to 1s?

Reply:
No longer changing scaleUp, but I am trying to remove any downtime in scale down. The helm repo seems to be using `kubernetes v1.23` and for some reason the scaleDown option is being ignored (and the default of 300s is used).  I now also added a timeout to at least unblock the helm tests from running. Do you have any thoughts on best way forward for https://github.com/open-telemetry/opentelemetry-operator/issues/1574?

Reply:
@moh-osman3 [they are using 1.25](https://github.com/open-telemetry/opentelemetry-helm-charts/actions/runs/4424164450/jobs/7757660240#step:3:117)